docs/src/InvariantTheory/reductive_groups.md

```diff
@@ -24,32 +29,57 @@ See also [DK15](@cite) and [DJ98](@cite).
 
 ## Creating Invariant Rings
 
-There are no exact means to handle algebraically closed fields on the computer. For the computation of invariant rings in the above setting, on the other hand, there is no need to deal with explicit elements of ``G`` or with its group structure. The implementation of Derksen's algorithm in OSCAR can  handle situations where both $G$ and the representation $\rho$ are defined over an exact subfield $k$ of $K$ which is supported by OSCAR: 
+### How Linearly Reductive Groups and Their Representations are Given
+
+There are no exact means to handle algebraically closed fields on the computer. For the computation of invariant rings in the above setting,
+on the other hand, there is no need to deal with explicit elements of ``G`` or with its group structure. The implementation of Derksen's
```

Comment:
That's not quite true... We have `algebraic_closure(QQ)` and the `QQBarField` it returns works quite well, and we are using it in parts of the book.

But of course we can't just use "the complex numbers" or something like that.

```suggestion
For the computation of invariant rings in the above setting, there is no need
to deal with explicit elements of ``G`` or with its group structure. The
implementation of Derksen's
```